src/deps/lol-html.zig

```diff
@@ -582,7 +582,7 @@ pub const HTMLString = extern struct {
 
     pub fn toString(this: HTMLString) bun.String {
         const bytes = this.slice();
-        if (bun.strings.isAllASCII(bytes)) {
+        if (bytes.len > 0 and bun.strings.isAllASCII(bytes)) {
```

Comment:
is this necessary?

Reply:
Yes, it is an assertion failure to use createExternal on a zero-length string